src/Resources/Files.php

```diff
@@ -2,16 +2,16 @@
 
 namespace SevenShores\Hubspot\Resources;
 
-class Files extends Resource
+class Files extends Endpoint
 {
     /**
      * Upload a new file.
      *
-     * @param resource|string $file
-     *
-     * @see https://legacydocs.hubspot.com/docs/methods/files/v3/upload_new_file
+     * @param Endpoint|string $file
```

Comment:
there is need to return initial code (`resource`)
Because `resource` (all letters are lower case) is default php type which returns `fopen` function
for this place 
`@param resource|string $file`

Reply:
I merged
I ll fix it myself

Reply:
Unfortunately, we are pressed for time, so I've decided to finish fixing this issue myself.
I've already merged it with your changes.
Thanks for your efforts!
@wilsonpinto